### PR TITLE
Added plays and currentplay

### PR DESCRIPTION
**Pull request by KCNilssen**
Adds the last two parts of the game class, allPlays and currentPlay

### CODE REVIEW

**Review by Mattsface — COMMENTED** [review]
👍

File: mlbStatsAPI/game.py [hunk 1]
```diff
@@ -141,6 +142,291 @@ class HitCoordinate:
...
+
+    def __repr__(self):
+        return str(self.description)
+
+    def asdict(self):
+        return asdict(self)
+
+
 @dataclass(frozen=True)
 class Hit_LD:
```
[the remark this answers is not in the document]
Comment (KCNilssen) [reply mlbStatsAPI/game.py]: liveData
    - plays
          - playsByInning
          
Holds the hits

[the remark this answers is not in the document]
Comment (KCNilssen) [reply mlbStatsAPI/game.py]: _LD = liveData

**Review by Mattsface — APPROVED** [review]
LGTM

**Pull request merged**